downstream/modules/platform/proc-create-controller-form-view.adoc

```diff
@@ -0,0 +1,19 @@
+[id="proc_create-controller-form-view_{context}"]
+
+= Create-{ControllerName}-form-view
+
+== Using the Form View
+
+. Ensure *Form view* is selected. It should be selected by default.
```

Comment:
Will the user be in the location where form view is selected? Should we tell them to navigate to a specific place in the UI first? Without an abstract and directions on where to navigate, this seems like it's missing something.j

Reply:
@jself-sudoku This is what the UI looks like there:

![image](https://user-images.githubusercontent.com/11698892/193657659-3a370951-3544-4146-b084-c84b229a70e1.png)

There is a toggle for switching between Form view, and Yaml view.  The default when loading the page is Form view.

Reply:
The directions are in another module. Something that probably should be addressed later.